soundfile.py

```diff
@@ -1333,7 +1341,10 @@ def _cdata_io(self, action, data, ctype, frames):
         frames = func(self._file, data, frames)
         _error_check(self._errorcode)
         if self.seekable():
-            self.seek(curr + frames, SEEK_SET)  # Update read & write position
+            if self.format != 'OGG':
+                # OGG(Vorbis, Opus) has seek bug from libopusfile
+                # see: https://github.com/libsndfile/libsndfile/issues/794
+                self.seek(curr + frames, SEEK_SET)  # Update read & write position
```

Comment:
It might be better to put this into `seekable()`, and just define OGG files as not-seekable.

Reply:
sounds good!